neqo-transport/src/connection/mod.rs

```diff
@@ -2932,6 +2932,21 @@ impl Connection {
         Ok(())
     }
 
+    pub fn stream_bytes_written(&mut self, stream_id: StreamId) -> Res<u64> {
```

Comment:
Doing three stream lookups like this might push me toward having the stats returned as a struct from here.   That is:

```rust
pub fn stream_stats(&self, stream_id: StreamId) -> Res<Option<SendStreamStats>> {
   Ok(self.streams.get_send_stream(stream_id).map(|s| s.stats()))
}
```

The webtransport code will need to unwrap this and tweak the values, but they will own the answer, so it should be OK.